utils/appError.js

```diff
@@ -0,0 +1,10 @@
+class AppError extends Error {
+    constructor(statusCode, status, message) {
+        super(message);
+        this.statusCode = statusCode;
+        this.status = status;
+        this.message = message;
```

Comment:
duplicate with 3 ?

Reply:
No. After check the message passed in the constructor is to add the message to the stack trace and avoid a duplication of the message. the this message is to have a visible error message to extract easily to later display to the user

Reply:
![image](https://user-images.githubusercontent.com/52856615/112290841-e4b1c680-8c8f-11eb-9ff5-888cdb5c951b.png)
Expected result